figures/views.py

```diff
@@ -357,8 +365,10 @@ class GeneralUserDataViewSet(CommonAuthMixin, viewsets.ReadOnlyModelViewSet):
     model = get_user_model()
     pagination_class = FiguresKiloPagination
     serializer_class = GeneralUserDataSerializer
-    filter_backends = (DjangoFilterBackend, )
+    filter_backends = (SearchFilter, DjangoFilterBackend, OrderingFilter)
     filter_class = UserFilterSet
+    search_fields = ['username', 'email', 'profile__name']
```

Comment:
```suggestion
    search_fields = ['username', 'email', 'name']
```

Reply:
I cannot change this,  the search fields names need to be for valid model fields, it cannot be serializer fields, so I need to use the look up.